src/test/java/seedu/address/logic/commands/EditPatientCommandTest.java

```diff
@@ -0,0 +1,152 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.commands.NewCommandTestUtil.DESC_AMY;
+import static seedu.address.logic.commands.NewCommandTestUtil.DESC_BOB;
+import static seedu.address.logic.commands.NewCommandTestUtil.VALID_NAME_BOB;
+import static seedu.address.logic.commands.NewCommandTestUtil.VALID_PHONE_BOB;
+import static seedu.address.logic.commands.NewCommandTestUtil.assertCommandFailure;
+import static seedu.address.logic.commands.NewCommandTestUtil.assertCommandSuccess;
+import static seedu.address.logic.commands.NewCommandTestUtil.showPersonAtIndex;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_PERSON;
+import static seedu.address.testutil.TypicalIndexes.INDEX_SECOND_PERSON;
+import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.logic.commands.EditPatientCommand.EditPersonDescriptor;
+import seedu.address.model.AddressBook;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.person.Person;
+import seedu.address.testutil.EditPersonDescriptorBuilder;
+import seedu.address.testutil.PersonBuilder;
+import seedu.address.testutil.TypicalPersons;
+
+/**
+ * Contains integration tests (interaction with the Model, UndoCommand and RedoCommand)
+ * and unit tests for EditPatientCommand.
+ */
+public class EditPatientCommandTest {
```

Comment:
Great work on writing the unit tests! For v1.1, we might not need it but getting it done earlier is a nice bonus.